pkg/platform/model/recipe.go

```diff
@@ -165,15 +173,21 @@ func fetchRecipeID(commitID strfmt.UUID, owner, project string, hostPlatform *st
 		}
 	}
 
-	// Because we filter platforms in the request we should only
-	// receive one recipe ID
-	if len(recipeID.Payload) != 1 {
-		return nil, FailOrderRecipes.New("err_recipe_payload")
+	if hostPlatform == nil {
+		hostPlatform = &HostPlatform
```

Comment:
I'd rather we rely on this always being passed in, so we're not making this function rely on globals.